website/project/views/contributor.py

```diff
@@ -669,6 +670,12 @@ def claim_user_form(auth, **kwargs):
     user.update_guessed_names()
     # The email can be the original referrer email if no claimer email has been specified.
     claimer_email = unclaimed_record.get('claimer_email') or unclaimed_record.get('email')
+
+    # If there is a registered user with this email, redirect to 're-enter password' page
+    user_from_email = User.find_one(Q('username', 'eq', claimer_email))
```

Comment:
@cslzchen  So what it looks like to me based on the [blame](https://github.com/CenterForOpenScience/osf.io/blame/develop/website/project/views/contributor.py#L674) is that they're both in there but [540aaac](https://github.com/CenterForOpenScience/osf.io/pull/6362/commits/540aaac5a719c8e099cb97489b58a5e16a8e7502) is overwriting [b885376](https://github.com/CenterForOpenScience/osf.io/pull/6362/commits/b885376a04cd40ad995716ea04536a06d1819350) because it came later, which tracks. Is there a problem with how it is right now? @caseyrollins is going to get with you to see if there needs to be changes.